budgetportal/tests/test_provincial_infrastructure_projects.py

```diff
@@ -1,5 +1,9 @@
 import os
+import random
```

Comment:
remove now it's unused

Reply:
removed

https://github.com/vulekamali/datamanager/pull/190/commits/21ead8374fd5b19678775974b76ca83037aad7ba